jabel-javac-plugin/src/main/java/com/github/bsideup/jabel/JabelCompilerPlugin.java

```diff
@@ -27,7 +28,17 @@
 
     @Override
     public void init(JavacTask task, String... args) {
-        Instrumentation instrumentation = ByteBuddyAgent.install();
+        Instrumentation instrumentation = ByteBuddyAgent.install(
+                new AttachmentProvider.Compound(
+                        AttachmentProvider.ForJ9Vm.INSTANCE,
+                        AttachmentProvider.ForStandardToolsJarVm.JVM_ROOT,
+                        AttachmentProvider.ForStandardToolsJarVm.JDK_ROOT,
+                        AttachmentProvider.ForStandardToolsJarVm.MACINTOSH,
+                        AttachmentProvider.ForUserDefinedToolsJar.INSTANCE,
+                        AttachmentProvider.ForEmulatedAttachment.INSTANCE,
+                        AttachmentProvider.DEFAULT
```

Comment:
This last one seems redundant. Rather add the provider for the modularized VM.

Reply:
Thanks! The idea is to fallback to the default, so that if you add more providers, they will be taken into an account without Jabel having to sync with them :)